rules/integrations/cloud_defend/container_workload_protection.toml

```diff
@@ -22,6 +22,10 @@ name = "Container Workload Protection"
 risk_score = 47
 rule_id = "4b4e9c99-27ea-4621-95c8-82341bc6e512"
 rule_name_override = "message"
+setup = """## Setup
+
+The `max_signals` field is set to a value greater than the default value (1000) set by `system_limit`. This is to ensure that all alerts are captured.
+To bypass this default, configure the `xpack.alerting.rules.run.alerts.max` setting in the Kibana config."""
```

Comment:
The term `max_signals` can be unknown to UI-only users that don't interact with the API directly. I think we can steal some ideas from our [docs](https://github.com/elastic/security-docs/pull/5106/files) to make the setup guide more user-friendly:

```suggestion
setup = """## Setup

### Max alerts per run

This rule is configured to generate more **Max alerts per run** than the default 1000 alerts per run set for all rules. This is to ensure that it captures as many alerts as possible.

**IMPORTANT:** The rule's **Max alerts per run** setting can be superseded by the `xpack.alerting.rules.run.alerts.max` Kibana config setting, which determines the maximum alerts generated by _any_ rule in the Kibana alerting framework. For example, if `xpack.alerting.rules.run.alerts.max` is set to 1000, this rule will still generate no more than 1000 alerts even if its own **Max alerts per run** is set higher.

To make sure this rule can generate as many alerts as it's configured in its own **Max alerts per run** setting, increase the `xpack.alerting.rules.run.alerts.max` system setting accordingly.

**NOTE:** Changing `xpack.alerting.rules.run.alerts.max` is not possible in Serverless projects."""
```

Reply:
LGTM, thanks for the suggestion, @banderror !

Reply:
Added!